Fix: Re-writed code with bug fixes

Barrandov opened the pull request:
https://rawgit.com/Barrandov/fake-burrberry-frontend/feat/product-page/product-page.html

evgenyrodionov commented:
Коммиты не по формату

evgenyrodionov commented:
Контент должен тянуться, у тебя фикс ширина это плохо
![image](https://user-images.githubusercontent.com/775692/28960175-45580450-7906-11e7-9132-850f1397c0b5.png)

блок more for you должен стоять под карточками
![image](https://user-images.githubusercontent.com/775692/28960172-42342614-7906-11e7-8dc5-9e2afb0b0d85.png)

читай https://medium.com/@evgeny.rodionov/feb04a46a8fa

evgenyrodionov commented:
Проверю в 3

Barrandov commented:
FEAT: Major Update: Added apaptive version (works both on tablet and mobile).
FEAT: Added responsive

FIX : кодстайл в хтмл
FIX : a11y
FIX : кнопки
FIX : иконки в разметке
FIX : проблемы в описании
FIX : нейминг
FIX : кодстайл в цсс

Barrandov commented:
*** В рекоменд картинах я выставил хайт 174px чтоб по зепелину нормально смотрелось. Там же написал коммент что только для макета.

evgenyrodionov commented:
Коммиты должны быть по формату

evgenyrodionov commented:
Картинке одиноко без текста

![image](https://user-images.githubusercontent.com/775692/28997631-f0826200-7a21-11e7-9c7b-ac1e7c4f26b8.png)

evgenyrodionov commented on css/style.css:
```diff
@@ -1,331 +1,591 @@
-  white-space: nowrap;
+@media only screen and (min-width: 48rem) {
+  .product__title {
+    margin: 1.5rem 0rem 1.5rem 0rem;
```
сокращай

evgenyrodionov commented on css/style.css:
```diff
@@ -1,331 +1,591 @@
 }
 
-.line-short {
+.xs-width {
```
неочевидный нейминг

evgenyrodionov commented on css/style.css:
```diff
@@ -1,331 +1,591 @@
+  }
+}
 
+.product__operations {
```
actions мб

evgenyrodionov commented on product-page.html:
```diff
@@ -1,126 +1,221 @@
+    <hr class="full-width">
+
+    <section class="product__related">
+        <h2 class="product__related-title">WE RECOMMEND</h2>
```
откуда 4 пробела взялось? поставь себе какой-нибудь бьютифайер

evgenyrodionov commented on product-page.html:
```diff
@@ -1,126 +1,221 @@
+          <div class="row">
+            <div class="col-md-3">
+              <h4 class="footer-menu__title">CUSTOMER SERVICE</h4>
+              <a class="footer-menu__link" href="#">Contact Us</a>
```
в нав

Barrandov commented:
Женя, прошу прощения, коммиты опять не по формату. Ребята помогли разобраться до конца с гитом, впредь коммитеть буду по правилам. 

fix: Minify CSS margins
fix: Change name in some classes
fix: Make HTML more readable
fix: A11Y compatibility
feat: Add <nav> to footer
feat: Add role to appropriate classes

evgenyrodionov commented:
Добавь ссылку в логотип

evgenyrodionov commented:
гит чекаут мастер, гит пулл ориджин мастер --ребейз, новые бранчи делаешь от обновленного мастера
